scala-api/src/test/scala/com/cloudera/livy/ScalaTestClient.scala

```diff
@@ -21,275 +21,241 @@ import java.io._
 import java.net.URI
 import java.nio.charset.StandardCharsets._
 import java.util._
-import java.util.concurrent.TimeUnit
 import java.util.jar.JarOutputStream
 import java.util.zip.ZipEntry
 
-import com.cloudera.livy.client.LivyScalaClient
 import com.cloudera.livy.rsc.RSCConf.Entry._
 import org.apache.spark.SparkFiles
 import org.apache.spark.launcher.SparkLauncher
 import org.mockito.Mockito._
-import org.scalatest.FunSpecLike
+import org.scalatest.FunSuite
 import org.scalatest.concurrent.ScalaFutures
 
 import scala.collection.mutable.ArrayBuffer
 import scala.concurrent.Await
 import scala.concurrent.duration._
 import scala.language.postfixOps
+import scala.util.Success
+import scala.util.Failure
+import scala.concurrent.ExecutionContext.Implicits.global
 
-class ScalaTestClient extends FunSpecLike with ScalaFutures {
+class ScalaTestClient extends FunSuite with ScalaFutures {
```

Comment:
This file and class should be called either `TestScalaClient` or `ScalaClientTest` to follow the convention.